029-IoTEdge/Student/Challenge-06.md

```diff
@@ -0,0 +1,64 @@
+# Challenge 6: Deploy to devices at scale
+
+[< Previous Challenge](./Challenge-05.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-07.md)
+
+## Pre-requisites
```

Comment:
Again, please delete this section from all challenge text, it's implied they are done in order.